### PR TITLE
appveyor: move install instruction to CMakeLists.txt

### PR DESCRIPTION
**Pull request by kenhys**
It is better way to do so.

### CODE REVIEW

**Comment by kenhys** [comment]
It works as expected.
https://ci.appveyor.com/project/kenhys/mroonga/builds/25244620

File: CMakeLists.txt [hunk 1]
```diff
@@ -471,3 +471,8 @@ install(FILES
   "${PROJECT_BINARY_DIR}/data/install.sql"
   "${PROJECT_SOURCE_DIR}/data/uninstall.sql"
   DESTINATION "${MRN_DATA_DIR}/")
+if(MSVC AND CMAKE_VERSION VERSION_GREATER "3.1")
+  install(FILES $<TARGET_PDB_FILE:mroonga>
```
Comment (komainu8) [inline CMakeLists.txt]: Where "TARGET_PDB_FILE:mroonga" is defined?
I don't find that definition.

Comment (kenhys) [reply CMakeLists.txt]: @komainu8 

TARGET_PDB_FILE: is CMake builtin feature.
https://cmake.org/cmake/help/v3.1/manual/cmake-generator-expressions.7.html#informational-expressions
So we use `$<TARGET_PDB_FILE:PROJECT_NAME>` is used in here.

Comment (komainu8) [reply CMakeLists.txt]: I see. Thanks!

**Pull request merged**